### ISSUE TITLE
Identifier has already been declared

### ISSUE THREAD

**Issue by kenneho** [CarterGrimmeisen/zod-prisma#27]
Hi.

My generated zod file have suddenly started getting errors like this: 

`Syntax error: Identifier 'RelatedCommentModel' has already been declared`

Extract from the file itself:

```
import { CompleteComment, RelatedCommentModel } from "./index"

export const CommentModel = z.object({ ....

export interface CompleteComment extends Comment { ....

export const RelatedCommentModel: z.ZodSchema<CompleteComment> = z.lazy(() => CommentModel.extend({  ....
```
I may be because my model references itself, i.e. the `Comment` model references other comments.

**Comment by CarterGrimmeisen** [comment]
Ah, okay. I hadn't considered this use case. For now you can manually remove the imports but I will get a fix for this out in the next few days! Thanks for the report.

**Comment by CarterGrimmeisen** [comment]
This has been resolved and a new version published `0.2.2`. Thanks again for the report!

**Comment by kenneho** [comment]
Thanks for providing a fix, and thanks again for this great library!
